addons/point_of_sale/static/src/app/components/popups/closing_popup/closing_popup.js

```diff
@@ -191,6 +191,21 @@ export class ClosePosPopup extends Component {
     }
     async closeSession() {
         this.pos._resetConnectedCashier();
+
+        const draftOrders = this.pos.models["pos.order"].filter(
+            (o) => o.state === "draft" && typeof o.id === "string" && o.lines.length > 0
```

Comment:
```suggestion
            (o) => o.state === "draft" && this.isSynced && o.lines.length
```

you can use get isSynced() here